QUARKUS-280.md

```diff
@@ -0,0 +1,31 @@
+## QUARKUS-280 - Tech Preview support for Message tracing and Open Telemetry
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-280
+
```

Comment:
Add link to the analysis done on RFE - https://issues.redhat.com/browse/QUARKUS-280?focusedCommentId=18936836&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-18936836